### PR TITLE
Trivia: Fix translation formatting

### PR DESCRIPTION
**Pull request by AnnikaCodes**
This also exempts `tr` from `eslint`'s line length checking, just like `Utils.html`.

### CODE REVIEW

File: .eslintrc-no-types.json [hunk 1]
```diff
@@ -218,7 +218,7 @@
         "max-len": ["warn", {
           "code": 120, "tabWidth": 0,
           // see bottom of file for source
-          "ignorePattern": "^\\s*(?:\\/\\/ \\s*)?(?:(?:export )?(?:let |const )?[a-zA-Z0-9_$.]+(?: \\+?=>? )|[a-zA-Z0-9$]+: \\[?|(?:return |throw )?(?:new )?(?:[a-zA-Z0-9$.]+\\()?)?(?:Utils\\.html|\\$\\()?['\"`/]"
+          "ignorePattern": "^\\s*(?:\\/\\/ \\s*)?(?:(?:export )?(?:let |const )?[a-zA-Z0-9_$.]+(?: \\+?=>? )|[a-zA-Z0-9$]+: \\[?|(?:return |throw )?(?:new )?(?:[a-zA-Z0-9$.]+\\()?)?(?:Utils\\.html|\\$\\()?(?:(this\\.)?(room\\.)?tr|\\$\\()?['\"`/]"
```
Comment (Zarel) [inline .eslintrc-no-types.json:221]: This is compiled code. You should be editing the source regex on line 304. The compiler is here: https://zarel.github.io/regexfree/

Comment (AnnikaCodes) [reply .eslintrc-no-types.json:221]: Oh! Sure, I'll do that.

File: .eslintrc-no-types.json [hunk 2]
```diff
@@ -323,11 +323,12 @@ REGEXFREE SOURCE FOR IGNOREPATTERN: https://zarel.github.io/regexfree/
...
   (
     Utils\.html
+  |
+    (this\.)?(room\.)?tr
   |
     \$\(
   )?
 
   # start of string or regex
   ['"`\/]
```
Comment (Zarel) [inline .eslintrc-no-types.json:333]:
```suggestion
  ['"`\/]

```

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**